src/Claims/DistributedParser.php

```diff
@@ -47,7 +47,7 @@ public function fetch(OpenIDClient $client, array $claims, array $accessTokens =
         $claimSources = $claims['_claim_sources'] ?? null;
         $claimNames = $claims['_claim_names'] ?? null;
 
-        if (! is_array($claimSources)) {
+        if (! array_key_exists('_claim_sources', $claims) || ! is_array($claimSources)) {
```

Comment:
Why this new check? If _claim_sources does not exist it will be null as for the check 3 lines before